.github/workflows/kuzzle-core.yml

```diff
@@ -0,0 +1,42 @@
+name: kuzzleio/kuzzle
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  dockerhub-deploy:
+    name: Kuzzle core Node.js
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout project
+        uses: actions/checkout@v3
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+
+      - name: Get current version from package.json
+        shell: bash
+        id: get-version
+        run: |
+          echo "version=$(jq -r .version package.json)" >> $GITHUB_OUTPUT
+          echo "major-version=$(jq -r .version package.json | cut -d. -f 1)" >> $GITHUB_OUTPUT
+
+      - name: Build and push
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          file: ./docker/images/kuzzle/Dockerfile
+          push: true
+          platforms: "linux/amd64,linux/arm64"
+          tags: kuzzleio/kuzzle:${{ steps.get-version.outputs.major-version }},kuzzleio/kuzzle:latest,kuzzleio/kuzzle:${{ steps.get-version.outputs.version }}
```

Comment:
https://github.com/docker/metadata-action could be interesting to use, because it autogenerate tags and opencontainers labels.

Reply:
It would, I'll keep this in mind for the futur implementation of semantic release